src/main/resources/sql/V1__basic_tables.sql

```diff
@@ -0,0 +1,64 @@
+CREATE SCHEMA IF NOT EXISTS messaging;
```

Comment:
The `V1__basic_tables.sql` name have extra underscore in first case.
Do we need it by convention? If not, let's remove it.

[Optional] Also, I propouse to start a description from the verb to make it more reasonable.
For example, it might look like this - `V1_add_basic_tables.sql`

Reply:
@AlekseiLitkovetc 
about the extra underscore - is the default parameter, but it's configurable. I agree it's better with one)

Reply:
corrected